airbyte-server/src/main/java/io/airbyte/server/handlers/DestinationDefinitionsHandler.java

```diff
@@ -179,6 +187,10 @@ public DestinationDefinitionRead createPrivateDestinationDefinition(final Destin
     final StandardDestinationDefinition destinationDefinition = destinationDefinitionFromCreate(destinationDefCreate)
         .withPublic(false)
         .withCustom(false);
+    if (!protocolVersionRange.isSupported(new Version(destinationDefinition.getProtocolVersion()))) {
+      throw new RuntimeException(String.format("Airbyte Protocol Version %s is not supported. (Must be within [%s:%s])",
```

Comment:
Should this be replaced with a specific exception?  Similar to the `IdNotFoundKnownException` exception thrown above?  It should made this code a little less verbose as the exception itself could take the version objects here and generate the message within itself.

  (This question applies to all the `RuntimeException`s being thrown in this PR.)

Reply:
Good point. Also discovered that `KnownException` comes with the logic to define the error code.
Updated.